vignettes/intro.Rmd

```diff
@@ -27,14 +29,31 @@ require(sessioninfo)
 
 This package is an R shiny implementation of the [**Gosling.js**](http://gosling-lang.org/) library. Gosling js is a highly *expressive* library based on a *Grammar for scalable and interactive Genomics Data visualization*. This library is build upon the [**React** wrapper of the Gosling.js](https://github.com/gosling-lang/gosling-react). Which is powered by [**Shiny.React**](https://github.com/gosling-lang/gosling-react). Thus most of the Charts can be directly build using this package.
 
-Let's start with an example. For that let's read a csv file which has some Genomic data with chromosome start stop indicators. This dataset is directly picked from github [here](https://github.com/sehilyi/gemini-datasets). You can read more about the dataset by going to there website [here](https://gemini.readthedocs.io/en/latest/index.html). In short GEMINI (Genome MINIng) is a flexible framework for exploring genetic variation in the context of the wealth of genome annotations available for the human genome.
+Let's start with an example. For that let's read a csv file which has some Genomic data with chromosome start stop indicators. This dataset is directly picked from gosling-lang.org [here](https://server.gosling-lang.org/api/v1/tileset_info/?d=cistrome-multivec).
 
-```{r store-data-links, eval=FALSE}
-gemini_github_path <- "https://rb.gy/ezv5k"
+"Multivec" is a term used in genomics to refer to a specific type of data format used for representing and visualizing multi-dimensional numerical data across genomic coordinates. It's commonly used for representing data like ChIP-seq, ATAC-seq, Hi-C, and other genomic experiments where signals or measurements are collected at various genomic positions.
```

Comment:
See comment below about this paragraph (in linechart.rmd)

Reply:
Updated.